CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -85,13 +88,20 @@ function Panel3D.Get(panelId)
 
 		panel.transparencyCallbacks = {}
 
+		panel.OnMouseEnter = false
+		panel.OnMouseLeave = false
```

Comment:
Since they are the same value why don't we do something like.
`panel.OnMouseEnter, panel.OnMouseLeave = false`
They will still act like to different variables, but since they have the starting value it can save lines of code like SolarCrane said. Thoughts?